cmds/exp/currentBoot/main.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2020 the Gprovision Authors. All Rights Reserved.
```

Comment:
Just realized I hadn't updated the copyright line in this file. Before I bother, any thoughts on whether this is useful as a stand-alone executable or if it should be example(s) in the documentation?

Reply:
I'd probably rename it bootvars or something? (definitely without camel case)

but it seems very useful, yeah i'd keep it here.

Reply:
should this be limited only to boot vars? What about `efivars` which is more generic?

Reply:
Renaming it to bootvars doesn't stray too far from the current functionality, but `efivars` implies it can handle many different vars; unless greatly expanded, it can't handle anything but boot vars.

Reply:
sounds good. Any expansion of functionalities can be handled later